### PR TITLE
Clarify relationship bx securing mechanisms and media types

### LINKED ISSUE #1065: Media types do not indicate securing mechanism
_Originally posted by @Sakurann in https://github.com/w3c/vc-data-model/pull/1062#discussion_r1136323602_

>> media types do not indicate proofing mechanisms

>  I think we should apply the same philosophy to all media types related to this specification that media types describe a secured 'thing' and media type itself does not imply the proofing mechanism. given the base media type is application/vc+ld+json; all extensions of VCDM core spec should use `vc+`.

### PR DESCRIPTION
**Pull request by brentzundel**
This PR attempts to add some clarity around the relationship between securing mechanisms and media types.

It primarily uses language suggested in Issue #1065. 
It adds that media types defined that allow for transformation into application/vc+ld+json should begin with application/vc+
It states that application/vc+ld+json does not imply any securing mechanism
it states that other media types should not imply any securing mechanism
I removes the note about Issue #1065

fixes #1065

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/1107.html" title="Last updated on May 4, 2023, 9:48 PM UTC (1e350e3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1107/c964156...brentzundel:1e350e3.html" title="Last updated on May 4, 2023, 9:48 PM UTC (1e350e3)">Diff</a>

### CODE REVIEW

File: index.html
Comment (msporny) [inline index.html:3405]: I had to read this last sentence a number of times and I'm still not sure I understand what it's saying.

I think it's saying: "This core spec doesn't imply a securing mechanism, and other media types shouldn't imply a securing mechanism either"... 

IOW, "that other thing might have `proof` and `experimentalSecuringMechanism` and be wrapped in a CWT (or any combination thereof), so... you can't presume any of those things... you're just going to have to verify it using your own checks to see if it truly is verifiable, and then validate it to make sure it meets your business requirements."

I don't know how to say that more succinctly... once we have confirmation on the intent, we can probably go from there.

As an aside, it sounds like we're starting to collect: "Normative statements for specification authors that are authoring other specifications that contain transformation rules to `application/vc+ld+json`."

File: index.html
Comment (msporny) [inline index.html:3399]: I think this is good. I was wondering if we should say anything at all about this... the upside of not saying something is that other media types like `application/foo` could have a section about transformation into a VC w/o having to register two media types;  `application/foo` and  `application/vc+foo`... maybe the argument there is that the thing that's important is the transformation rules, not the media type? Does differentiating it via a new media type always help?

I'm fine w/ the text above for now, and it is a `SHOULD` so, people that are working on `application/foo` can choose to ignore it because they're concerned about media type proliferation. If they ignore the advice, my expectation is that we'd still add them to the VC Specifications Directory.

To be clear, this isn't a change request, just noting thoughts hoping others will engage in a discussion on the benefits/drawbacks of the specification requirement.

**Review by msporny — APPROVED** [review]

File: index.html
Comment (TallTed) [reply index.html:3399]: I'm a bit concerned that talking this way about `application/vc+` suggests that there's a single way to deal with any and all such media types, though the way to deal with a media type is most simply determined by the outer edges of the media type -- e.g., `application/vc+ld+json` is most simply handled as `application/json`, not as `application/vc+`.

File: index.html
Comment (OR13) [reply index.html:3405]: I agree, I would remove this:

```
Other specifications
defining media types that begin with `application/vc+` SHOULD NOT define those
media types to imply any particular securing mechanism.Other specifications
defining media types that begin with `application/vc+` SHOULD NOT define those
media types to imply any particular securing mechanism.
```

A general statement about media types here might be:

Implementers are advised not assume veracity or security of content regardless of the media type used to communicate it. A validation process needs to be applied to confirm integrity, media types should not be trusted, or assumed to be sure without confirming by some validation process.

**Review by OR13 — CHANGES_REQUESTED** [review]
I think we should remove or refactor this sentence:

https://github.com/w3c/vc-data-model/pull/1107/files#r1180475199

**Comment by brentzundel** [comment]
> I think we should remove or refactor this sentence:
> 
> https://github.com/w3c/vc-data-model/pull/1107/files#r1180475199
> 
> The rest of the PR looks to be in pretty good shape.

This has been refactored in [0d5dbd9](https://github.com/w3c/vc-data-model/pull/1107/commits/0d5dbd985a84c27311cc86225a898904f115c1af)

File: index.html
Comment (brentzundel) [reply index.html]: This has been refactored in [0d5dbd9](https://github.com/w3c/vc-data-model/pull/1107/commits/0d5dbd985a84c27311cc86225a898904f115c1af)

File: index.html
Comment (brentzundel) [reply index.html]: This has been refactored in [0d5dbd9](https://github.com/w3c/vc-data-model/pull/1107/commits/0d5dbd985a84c27311cc86225a898904f115c1af)

File: index.html
Comment (andresuribe87) [inline index.html:3403]: This would make this statement more consistent with our terminology definitions. 
```suggestion
needs to be applied to enable verification of integrity. This also applies to
```

**Comment by brentzundel** [comment]
@OR13 please re-review and let me know if you would like any more changes

**Comment by msporny** [comment]
Waiting on review by @OR13 to merge.

**Review by andresuribe87 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

File: index.html
Comment (OR13) [inline index.html:3404]:
```suggestion
mechanism.  Do not
```

**Review by OR13 — CHANGES_REQUESTED** [review]
I agree with the intention of the PR, but I think it does not reflect the current normative requirements and their interpretation by developers.

This paragraph seems to contract dict existing requirements.

Perhaps a smaller paragraph can be made compatible with the existing requirements, and we can adjust / add more, if we are able to reach consensus on the related open PRs.

File: index.html
Comment (OR13) [reply index.html:3404]: "It describes an object that is securable, but a securing mechanism needs to be applied to enable verification of integrity." This sentence hurts... and I think we are doing a better job of addressing it in: 

- https://github.com/w3c/vc-data-model/pull/1101

File: index.html
Comment (brentzundel) [reply index.html:3401]: The language suggested in this PR is an attempt to clarify that in spite of the requirement to include a specific `@context`, and in spite of that `@context` containing references to a specific securing mechanism, no specific securing mechanism should be assumed. Just because the `@context` points to data integrity doesn't mean the data has been secured.

Happy for  further suggestions on how to better communicate that here.

File: index.html
Comment (brentzundel) [reply index.html]: I have reworked that sentence in an attempt to make it less painful in [1e350e3](https://github.com/w3c/vc-data-model/pull/1107/commits/1e350e3f328707fbb4ee16012ef4d3aff4a5e56a)

**Review by decentralgabe — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-05-10-vcwg#section1-5) on 2023-05-10

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.5. Clarify relationship bx securing mechanisms and media types (pr vc-data-model#1107)

_See github pull request [vc-data-model#1107](https://github.com/w3c/vc-data-model/pull/1107)._




**Manu Sporny:** PR 1107 - topic clarifying relationship between securing mechanisms and media types.  

**Orie Steele:** needs to review recent changes.  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
